docs/docs/policies.mdx

```diff
@@ -52,7 +52,7 @@ and Rule Policies might both predict with confidence 1), the priority of the
 policies is considered. Rasa policies have default priorities that are set to ensure the
 expected outcome in the case of a tie. They look like this, where higher numbers have higher priority:
 
-5. `RulePolicy`
+6. `RulePolicy`
```

Comment:
I think these numbers will be confusing for new 2.0 users who are not aware of old policies

Reply:
@erohmensing I think Vova has a point. Why did we remove the deprecated polices from this? I'm gonna merge my change anyway as this outside of the scope this PR imo.